EMS/client-helper-bundle/src/Helper/Templating/TemplateBuilder.php

```diff
@@ -71,4 +71,11 @@ public function isFresh(Environment $environment, TemplateName $templateName, in
 
         return $contentType->isLastPublishedAfterTime($time);
     }
+
+    public function exists(Environment $environment, TemplateName $templateName): bool
+    {
+        $settings = $this->settings($environment);
+
+        return null !== $environment->getLocal()->getTemplates($settings)->find($templateName->getSearchName());
```

Comment:
getLocal(), will throw an Exception if you are not in dev mode.

You should check with $environment->isLocalPulled.

Reply:
done